docs/en/ingest-management/fleet/add-fleet-server.asciidoc

```diff
@@ -2,46 +2,113 @@
 = Add a {fleet-server}
 
 To use {fleet} for central management, a <<fleet-server,{fleet-server}>> must
-be running and accessible to your hosts. This page describes how to add a
-{fleet-server} to an {ecloud} or self-managed deployment.
+be running and accessible to your hosts. There are a few approaches you can take
+when deploying {fleet-server}:
+
+* <<deployed-in-cloud>> as part of the hosted {ess}. Elastic manages both {fleet-server} and {es}.
+* <<deployed-on-prem>> to work with {es} running on-premises. You self-manage both {fleet-server} and {es}.
+* <<fleet-server-on-prem-es-cloud>>. You manage {fleet-server} and Elastic manages {es}.
+
+Read more about each to choose the best approach for your situation.
+
+[discrete]
+[[deployed-in-cloud]]
+== Provision {fleet-server} on {ecloud}
+
+// What is it...
+{fleet-server} can be provisioned and hosted on {ecloud}. In this case,
+when the deployment is created, a highly available set of {fleet-server}s
+is automatically deployed.
+
+// Benefits
+This approach might be right for you if you want to reduce on-prem compute,
+and you'd like Elastic to take care of provisioning and life cycle management of
+your deployment.
```

Comment:
>Feels like something is missing here: "reduce on-prem compute..." Does this mean reduce on-premises compute resources? costs?

@nimarezainia can you clarify your intent?

Reply:
yes you could say that. I was trying to say that literally reduces the compute resources required on the customer premises. Which does equate to cost - but there's more than cost of the actual compute - there's also the man power required to manage etc. This then quickly becomes a justification of on-premises vs Cloud. I would prefer to just keep it as is tbh, in other words it reduces amount of compute on-premises.

Reply:
The problem here is grammatical. "Compute" is a verb.  I think you just need to say "compute resources" so this sentence is not grammatically weird.